lib/ElementHandle.js

```diff
@@ -119,13 +119,14 @@ class ElementHandle extends JSHandle {
    * @return {!Promise<{x: number, y: number, width: number, height: number}>}
    */
   async boundingBox() {
-    const {model} = await this._client.send('DOM.getBoxModel', {
+    const result = await this._client.send('DOM.getBoxModel', {
       objectId: this._remoteObject.objectId
-    });
-    if (!model)
-      throw new Error('Node is detached from document');
+    }).catch(error => void debugError(error));
 
-    const quad = model.border;
+    if (!result)
+      throw new Error('Node is not visible');
```

Comment:
i'd return null in this case and throw in the `click` itself.

Reply:
Done.